lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2504,6 +2504,13 @@ createAcceleratorReceive(SILBuilder &B, SILLocation loc, SILType valueTy,
   return B.createBuiltin(loc, name, valueTy, subMap, operands);
 }
 
+static void
+verifyTensorComputationArgAsGuaranteedParam(FunctionRefInst *fnRef) {
+  const auto &conventions = fnRef->getConventions();
```

Comment:
Drop the `const` for local variables? The code here is pretty short and const is less idiomatic here.